x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/evaluate_panel.tsx

```diff
@@ -0,0 +1,321 @@
+          cellContentsElement,
+          children,
+        }: {
+          cellContentsElement: any;
+          children: any;
+        }) => {
+          const rowIndex = children?.props?.rowIndex;
+          const colId = children?.props?.columnId;
+          const gridItem = columnData[rowIndex];
+
+          if (gridItem !== undefined) {
+            const count = colId === gridItem.actual_class ? gridItem.count : gridItem.error_count;
+            return `${count} / ${gridItem.actual_class_doc_count} * 100 = ${cellContentsElement.textContent}`;
+          }
+
+          return cellContentsElement.textContent;
+        },
+      });
+    }
+  }, [confusionMatrixData]);
+
+  useEffect(() => {
+    const hasIsTrainingClause =
+      isResultsSearchBoolQuery(searchQuery) &&
+      searchQuery.bool.must.filter(
+        (clause: any) => clause.match && clause.match[`${resultsField}.is_training`] !== undefined
+      );
+    const isTrainingClause =
+      hasIsTrainingClause &&
+      hasIsTrainingClause[0] &&
+      hasIsTrainingClause[0].match[`${resultsField}.is_training`];
+
+    loadData({ isTrainingClause });
+  }, [JSON.stringify(searchQuery)]);
+
+  const renderCellValue = ({
+    rowIndex,
+    columnId,
+    setCellProps,
+  }: {
+    rowIndex: number;
+    columnId: string;
+    setCellProps: any;
+  }) => {
+    const cellValue = columnsData[rowIndex][columnId];
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    useEffect(() => {
+      setCellProps({
+        style: {
+          backgroundColor: `rgba(0, 179, 164, ${cellValue})`,
+        },
+      });
+    }, [rowIndex, columnId, setCellProps]);
+    return <span>{typeof cellValue === 'number' ? `${cellValue * 100}%` : cellValue}</span>;
+  };
+
+  if (isLoading === true) {
+    return <LoadingPanel />;
+  }
+
+  return (
+    <EuiPanel style={{ width: `${panelWidth}px` }}>
+      <EuiFlexGroup direction="column" gutterSize="s">
+        <EuiFlexItem>
+          <EuiFlexGroup gutterSize="s">
+            <EuiFlexItem grow={false}>
+              <EuiTitle size="xs">
+                <span>
+                  {i18n.translate(
+                    'xpack.ml.dataframe.analytics.classificationExploration.evaluateJobIdTitle',
+                    {
+                      defaultMessage: 'Evaluation of classification job ID {jobId}',
+                      values: { jobId: jobConfig.id },
+                    }
+                  )}
+                </span>
+              </EuiTitle>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <span>{getTaskStateBadge(jobStatus)}</span>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+        {error !== null && (
+          <EuiFlexItem grow={false}>
+            <ErrorCallout error={error} />
+          </EuiFlexItem>
+        )}
+        {error === null && (
+          <Fragment>
+            <EuiFlexItem grow={false}>
+              <EuiTitle size="xxs">
```

Comment:
I think this could do with an info tooltip, next to the Normalized confusion matrix title, to give a brief explanation as to what this is and what's it's showing, similar to what we have on the Regression page for the Mean Squared Error and R2.

![image](https://user-images.githubusercontent.com/7405507/70642039-7a117e80-1c36-11ea-9a21-a4c03aec9dfa.png)

Reply:
Good call - added a tooltip consistent with the language used in the docs 55ae0f3def4b9e8a33e63848272890d204f14d12